src/ApiSchema.ts

```diff
@@ -95,23 +103,34 @@ export class APISchema {
 
             const schema = await response.json();
 
-            const baseUrlRegex = this.constructBaseUrlRegex(schema);
-            const invalidBaseUrlRegex = this.constructInvalidBaseUrlRegex(schema);
-
+            this.platforms = schema.servers[0].variables.platform.enum;
             this.paths = [];
 
             for (const pathName in schema.paths) {
                 const pathSchema = schema.paths[pathName];
                 const methodSchema = pathSchema.get ? pathSchema.get : pathSchema.post;
 
                 if (!methodSchema) continue; // Only handle GET/POST
-                if (methodSchema.operationId.startsWith("tournament")) continue;
+                if (methodSchema.operationId.startsWith("tournament-v3")) continue;
 
                 const path = new Path();
```

Comment:
imo a fully fledged constructor instead of this building would be better suited here. You can just initialize the parameter maps to empty maps.